libs/ui/src/components/NeoButton/NeoButton.scss

```diff
@@ -120,6 +120,12 @@
   &.no-shadow {
     box-shadow: none;
   }
+
+  &--rounded {
+    border-radius: 9999px;
+    font-size: 0.75rem;
```

Comment:
Is it needed to change font-size here? What If I want to use rounded button with bigger font size